client/resource_group/controller/controller.go

```diff
@@ -91,14 +95,20 @@ type ResourceGroupsController struct {
 }
 
 // NewResourceGroupController returns a new ResourceGroupsController which impls ResourceGroupKVInterceptor
-func NewResourceGroupController(clientUniqueID uint64, provider ResourceGroupProvider, requestUnitConfig *RequestUnitConfig) (*ResourceGroupsController, error) {
-	// TODO: initialize `requestUnitConfig`` from the remote manager server.
-	var config *Config
-	if requestUnitConfig != nil {
-		config = generateConfig(requestUnitConfig)
-	} else {
-		config = DefaultConfig()
-	}
+func NewResourceGroupController(
+	ctx context.Context,
+	clientUniqueID uint64,
+	provider ResourceGroupProvider,
+	requestUnitConfig *RequestUnitConfig,
+) (*ResourceGroupsController, error) {
+	var err error
+	if requestUnitConfig == nil {
+		if requestUnitConfig, err = loadRequestUnitConfig(ctx, provider); err != nil {
+			return nil, err
+		}
+	}
+	// TODO: do we need to reload the config from remote server periodically?
```

Comment:
Do u mean we need watch it?

Reply:
Yes, but since the RU model config won't be modified frequently, I think it's ok to leave this as a nice-to-have option.

Reply:
Maybe we can consider adding an API to the triage reload. it's more "manual" and no need to restart the node.